src/test/java/hudson/plugins/clover/CloverWorkflowTest.java

```diff
@@ -12,6 +12,10 @@
 
 public class CloverWorkflowTest {
 
+    // BuildWatcher echoes job output to stderr as it arrives
+    @org.junit.ClassRule
```

Comment:
Use `import` for org.junit.ClassRule and org.jvnet.hudson.test.BuildWatcher, please.

Reply:
Done in a7af80737910b2c1f953540011f980c1d700981c